### DISCUSSION TITLE
Need to know process / binary initiating  connection and syncing on MAC

### DISCUSSION THREAD

**Discussion by razaat** [iterate-ch/cyberduck#15760] [Q&A]
We are evaluating Mountain Duck to use as a sync client for our SharePoint on-prem. We are using workspace one  Per-App tunnel to allow specific applications to connect to our enterprise environment. Would like to know  the process/binary initiating a connection and syncing for MD. 

Appreciate your support.

**Comment by tatze96** [comment]
The initiating process is Mountain Duck.exe (Windows) or, respectively, Mountain Duck.app (macOS), depending on the operating system.
